index.md

```diff
@@ -33,35 +33,38 @@ Here's a simple program:
 
 ```val
 subscript longer_of(_ a: inout String, _ b: inout String): String {
-  yield if b.count() > a.count() { &b } else { &a }
+  if b.count() > a.count() { yield &b } else { yield &a }
+}
+
+func emphasize(_ z: inout String, strength: Int = 1) {
+  z.append(repeatElement("!", count: strength)))
 }
 
 public fun main() {
   var (x, y) = ("Hi", "World")
-  inout z = longer_of[&x, &y]
-  z .append("!")
+  emphasize(longer_of[&x, &y])
```

Comment:
I'm afraid we need another another `&` here:
```val
emphasize(&longer_of[&x, &y])
```

Reply:
```suggestion
  emphasize(&longer_of[&x, &y])
```